### PR TITLE
Add german translation

### PR DESCRIPTION
**Pull request by Ztarbox**
It's my first GitHub collaboration and pull request so it's a small one and suggestions and tips are appreciated,

### CODE REVIEW

**Comment by Armaldio** [comment]
@Ztarbox Thanks for your contribution!
I can see 1 filed (https://github.com/Android-Messages-For-Desktop/android-messages-for-desktop/pull/18/files#diff-3a2d3d3dcecc6fffa8902ba0d2d100b5R13) that is not transalted, is it intended ?

@antoineguilbert Maybe reword the readme to redirect users to POEditor instead ?

**Comment by Ztarbox** [comment]
@Armaldio No. I just missed it. Thanks
